### ISSUE TITLE
profileInfo.isChild is always false

### ISSUE THREAD

**Issue by gruijter** [simplyGits/MagisterJS#89]
profileInfo.isChild is set to false, even if it is a child profile. This leads to a crash when doing m.children() while logged in as a pupil.

**Comment by lieuwex** [comment]
Does that crash provide any error details?

**Comment by gruijter** [comment]
It gives an error during the get here:
https://github.com/simplyGits/MagisterJS/blob/0cd4948bc4d888b2ad9a43724787636a5a4e54a7/src/magister.js#L208

**Comment by Argetan** [comment]
As far as I can tell `ProfileInfo.isChild` is constructed from the `Persoon` object in the account data only.
https://github.com/simplyGits/MagisterJS/blob/2087f489d6a850943d31e49932935c0259fb6062/src/magister.js#L416

`ProfileInfo.isChild` will then look for the `ZichtbaarVoorOuder` parameter.
https://github.com/simplyGits/MagisterJS/blob/2087f489d6a850943d31e49932935c0259fb6062/src/profileInfo.js#L72-L82

However, if I log-in on Magister and look at the requests it sends and the data it receives I don't see a `ZichtbaarVoorOuder` part in the `Persoon` JSON, even though I have set that, so it should be there. It is the case however that Magister makes another request to the following URL: `https://myschool.magister.net/api/leerlingen/XXXX/autorisatie` and receives the following JSON:
```
{
  "oudersMogenGegevensZien": true
}
```

I guess Magister changed the way they fetch that information. I don't know if that request is also being made if you are a parent and if so what the response looks like. I took a quick look at the privilliges as well and noticed one called `MataKinderen`, but I have no idea what that relates to (I have read access if that helps).
